packages/server/src/ApolloServer.ts

```diff
@@ -1121,7 +1143,7 @@ export class ApolloServer<in out TContext extends BaseContext = BaseContext> {
           : request.query,
     };
 
-    return await internalExecuteOperation(
+    const response: GraphQLResponse = await internalExecuteOperation(
```

Comment:
Can't `internalExecuteOperation` take a `TData` and not need this cast? If it's just simpler to stop the generics here I'd be amenable to that.

Reply:
I think we do need to have a cast *somewhere* right? Since we're not actually examining the data anywhere to make it the right thing?